tools/vagrant/README.md

```diff
@@ -168,24 +167,6 @@ vagrant ssh
 wskadmin user create <subject>
 ```
 
-This command will create a new *subject* with the authorization key shown on the console once you run `wskadmin`. This key is required when making API calls to OpenWhisk, or when using the command line interface (CLI). The namespace is the same as the `<subject>` name used to create the key.
```

Comment:
Looks like the same command is shown in the doc above. Seemed like the one removed was specific to namespaces.